### PR TITLE
Support flutter version `3.16.0`

### PR DESCRIPTION
**Pull request by aryan-more**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -15,8 +15,8 @@ dependencies:
 
   enough_html_editor:
     git:
-      url: https://github.com/linagora/enough_html_editor.git
-      ref: email_supported
+      url: https://github.com/aryan-more/enough_html_editor.git
+      ref: "patch-1"
```
Comment (hoangdat) [inline pubspec.yaml:19]: please wait for PR in `enough_html_editor` merge.

Comment (dab246) [reply pubspec.yaml:19]: We don't need to upgrade it

Comment (aryan-more) [reply pubspec.yaml:19]: @dab246 we need to upgrade that because enough_html_editor old package are not compatible with flutter 3.16.0

Comment (dab246) [reply pubspec.yaml:19]: I understand what you mean, but we only make changes for debugging purposes. When merging into the TMail project, we only need to upgrade `enough_html_editor`

Comment (aryan-more) [reply pubspec.yaml:19]: @dab246 yes i know that i will change the source of dependency as soon as enough_html_editor branch email-supported get merged to it.BTW i also changed flutter version to 3.16.0 in workflow ,so it workflow don't fail .

Comment (dab246) [inline pubspec.yaml:19]: @aryan-more We have merged the PR https://github.com/linagora/enough_html_editor/pull/24 into the `upgrade_v3.16.0` branch. Please use it to test on version `3.16.0`

Comment (aryan-more) [reply pubspec.yaml]: Hey @dab246 i have updated git repo in dependency to use upgrade_v3.16.0 branch

**Review by dab246 — APPROVED** [review]

**Pull request merged**